Source/common/SNTConfigurator.m

```diff
@@ -563,6 +568,14 @@ - (NSString *)bannedBlockMessage {
   return self.configState[kBannedBlockMessage];
 }
 
+- (NSString *)bannedUSBBlockMessage {
+  return self.configState[kBannedUSBBlockMessage];
```

Comment:
Comment in `SNTConfigurator.h` says a reasonable default is provided if not configured, but no default is provided.

Reply:
A little bit of indirection here that also holds true for the other parameters and block messages like this: the actual reasonable default is visible from [SNTDeviceMessageWindowController attributedCustomMessage]. 

I'm open to moving the default here and updating all of the other cases of this with other block messages all at once in a followup PR if that sounds better.

Reply:
I see that now that I got lower in the PR :)

I'd probably lean slightly more in favor of the default being provided by the configurator class just so its a common place to look for defaults.

As a slight tangent - I wonder if it makes sense for this message to be completely controlled. Meaning what if the current "default" messages are _**always**_ displayed. And then the configuration value is additional information. Could be useful if some organization wants to provide additional information to users such as a person to contact or policy information. Thoughts?

Reply:
The one case where I think this might fail is if the notification isn't in English, and the custom message needs to convey the same information but in another language.

I've moved this default to the configurator class to these modified keys. I'll do the same for all the other messages with defaults in a separate PR.